### DISCUSSION TITLE
RPI5 lightdm issue, RPI5 5GHZ Networks

### DISCUSSION THREAD

**Discussion by laurenttaieb** [RaspAP/raspap-webgui#1599] [Q&A]
Hi RaspAP team, I have a question, when installing RaspAP, lightdm doesn't restart after reboot. I have the same issues on RPI4 & RPI5, is there a known fix ?
Second Q, I noticed that 5GHZ network can't be activated on RPI4 unless US config is used. Does someone know if the same issue occurs on RPI5 , I'm trying to achieve a wireless connection over 50Mbps to enable good streaming.
Thanks

**Comment by billz** [comment]
>  lightdm doesn't restart after reboot

The lightdm service is installed with Desktop; this project [supports Lite distros only](https://docs.raspap.com/#compatible-operating-systems).

>  5GHZ network can't be activated on RPI4 unless US config is used

Untrue. Please read the docs. This is the [first search result for '5 GHz'](https://docs.raspap.com/ap-basics/#80211ac-5-ghz).

**Reply by laurenttaieb** [reply]
Thanks Billz, I'll go through the doc.

**Reply by breydan949** [reply]
> > lightdm doesn't restart after reboot
> 
> The lightdm service is installed with Desktop; this project [supports Lite distros only](https://docs.raspap.com/#compatible-operating-systems).
> 
> > 5GHZ network can't be activated on RPI4 unless US config is used
> 
> Untrue. Please read the docs. This is the [first search result for '5 GHz'](https://docs.raspap.com/ap-basics/#80211ac-5-ghz).

Hi, I just installed RaspAp in a clean Raspberry Pi 5 with Raspberry Pi Desktop distro, and base on the RaspAp documentation this system should be compatible, unless there is something that I'm misstating. However the problem I have is that after RaspAp installation (manual) OS Desktop UI stopped working, is this expected behavior or is there a workaround?

Thanks in advance
